frontend-react/src/components/FileHandlers/FileHandlerMessaging.tsx

```diff
@@ -220,7 +220,11 @@ const ErrorRow = ({ error, index }: ErrorRowProps) => {
     const { message, field, errorCode, trackingIds } = error;
     return (
         <tr key={"error_" + index}>
-            <td>{errorCode ? ErrorCodeTranslation[errorCode] : message}</td>
+            <td>
+                {errorCode && errorCode !== "UNKNOWN"
```

Comment:
Actually, I'll look into fixing the "blank if nothing is found" so when this gets merged we won't be break when there is an errorCode that Experience did not map a msg to.

Reply:
Okay, I added a 3rd condition in the if to make sure the returned value from the map is not empty, that fixes the "blank if nothing is found" because it will use the backend generated message instead if that is the case. That said, Experience should still look into overhauling this code to pull from a localization file.